devtools/cli/src/main/java/io/quarkus/cli/Create.java

```diff
@@ -0,0 +1,140 @@
+package io.quarkus.cli;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.Callable;
+
+import io.quarkus.cli.core.BaseSubCommand;
+import io.quarkus.devtools.commands.CreateProject;
+import io.quarkus.devtools.project.BuildTool;
+import io.quarkus.devtools.project.codegen.SourceType;
+import io.quarkus.platform.tools.config.QuarkusPlatformConfig;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "create", sortOptions = false, usageHelpAutoWidth = true, mixinStandardHelpOptions = false, description = "Create a new quarkus project.")
+public class Create extends BaseSubCommand implements Callable<Integer> {
+
+    @CommandLine.Option(names = { "-g",
+            "--group-id" }, order = 1, paramLabel = "GROUP-ID", description = "The groupId for project")
+    String groupId = "org.acme";
+
+    @CommandLine.Option(names = { "-a",
+            "--artifact-id" }, order = 2, paramLabel = "ARTIFACT-ID", description = "The artifactId for project")
+    String artifactId = "my-project";
+
+    @CommandLine.Option(names = { "-v",
+            "--version" }, order = 3, paramLabel = "VERSION", description = "The version for project")
+    String version = "1.0-SNAPSHOT";
+
+    @CommandLine.Option(names = { "-0",
+            "--empty" }, order = 4, description = "Generate without example code.")
+    boolean empty = false;
```

Comment:
@maxandersen didn't we say we should use `no-examples` in the end?

Reply:
Fine to change this, but I'm gonna kill somebody if I have to spend another week trying to get CI to pass over such a small change.

Reply:
@patriot1burke I can understand this 😂, it shouldn't break CI though :) let's wait for @maxandersen confirmation first.. 

This is not "that" small because it actually changes the spec of the CLI.. ex: if you give an example in a blog post and it changes in the next release, it's not great 😅

Reply:
Well, you guys have already changed this flag 3 times in the last two weeks and , well, i don't like any of your changes :)  So, if you change your mind yet again in the next two weeks, feel free to do the change yourself.

Reply:
i wasn't aware we were changing them here as those things were stil fluid.

lets just get this in and we can clean up / align these rather than do another round.